aea/helpers/base.py

```diff
@@ -45,15 +45,11 @@
 logger = logging.getLogger(__name__)
 
 
-def yaml_load(stream: TextIO) -> Dict[str, str]:
-    """
-    Load a yaml from a file pointer in an ordered way.
-
-    :param stream: the file pointer
-    :return: the yaml
-    """
+def _ordered_loading(fun: Callable):
```

Comment:
this is just a refactoring of previous helper functions, to reuse the code for `yaml_(load|dump)_all` helpers